api/controllers/mentee.js

```diff
@@ -0,0 +1,70 @@
+/* eslint-disable no-dupe-keys */
+const models = require('../../database/models');
+const response = require('../helpers/response');
+
+module.exports = {
+  async getAllMentees(req, res) {
+    try {
+      const mentees = await models.Mentees.findAll({
+        attributes: ['user_id', 'industry_id', 'location_id'],
+        include: [
+          {
+            model: models.Industries,
+            attributes: ['industry_name'],
+            as: 'industry'
+          },
+          {
+            model: models.Users,
+            attributes: [
+              'first_name',
+              'last_name',
+              'biography',
+              'profile_picture'
+            ],
+            as: 'user',
+            required: true,
+            include: [
+              {
+                model: models.Locations,
+                attributes: ['country_name', 'city_name'],
+                as: 'location'
+              }
+            ]
+          }
+        ]
+      });
+      if (mentees) {
+        return response.success(res, 200, mentees);
+      }
+      return response.error(res, 404, 'Could not find all Mentees');
```

Comment:
I would suggest changing the message to something along the line "No mentee found". Because the 'all' could imply that some mentees were returned and some weren't.